module/Application/src/Controller/FeedbackController.php

```diff
@@ -60,8 +60,10 @@ public function getList()
             );
         }
 
-        $from   = new DateTime($query['from']);
-        $to     = new DateTime($query['to']);
+        //  Create the date limits and ensure that the to date is adjsuted to the end of the day
+        $from = new DateTime($query['from']);
+        $to = new DateTime($query['to']);
+        $to->setTime(23, 59, 59);
```

Comment:
Not a blocker to merging, but you can add microseconds to this if you want.

Reply:
Have added the microseconds and pushed a new commit